packages/dds/sequence/src/revertibles.ts

```diff
@@ -545,7 +546,16 @@ function revertLocalSequenceRemove(
 				restoredRanges,
 				sharedString,
 			);
-			if (newStart !== undefined || newEnd !== undefined) {
+			// only move interval if start <= end
+			if (
```

Comment:
might be nice if you could do something like `(newStart ?? sharedString.localReferencePositionToPosition(interval.start)) <= (newEnd ?? sharedString.localReferencePositionToPosition(interval.end))`. but it's not obvious to me that at least one of newStart/newEnd is undefined. if not, I think this is better anyway.

Reply:
They could both be undefined.